cloudflare_app.py

```
@@ -0,0 +1,364 @@
+            'records': {},
+            'cloudflare_compatible': True
+        }
+        
+        try:
+            # فحص سجلات TXT
+            txt_records = []
+            try:
+                answers = dns.resolver.resolve(domain, 'TXT')
+                for rdata in answers:
+                    txt_records.append(str(rdata).strip('"'))
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)